docs/integrations/angular.rst

```diff
@@ -0,0 +1,48 @@
+Backbone
+========
+
+Installation
+------------
+
+Start by adding the ``raven.js`` script tag to your page. It should go **after** 
```

Comment:
This doesn't really matter now since things are patched when calling `install()`.